modules/library/referencing/src/main/java/org/geotools/referencing/cs/DefaultCoordinateSystemAxis.java

```diff
@@ -912,6 +912,18 @@ public static AxisDirection getDirection(String direction) throws NoSuchElementE
         if (candidate != null) {
             return candidate;
         }
+
+        /*
+         * In latest version of the database north-east and north-west are now northeast and northwest respectively,
+         * and none of these heuristics pick that up, so we add a few special cases here.
+         */
+        if ("northeast".equalsIgnoreCase(direction)) {
+            return AxisDirection.NORTH_EAST;
+        }
+        if ("northwest".equalsIgnoreCase(direction)) {
+            return AxisDirection.NORTH_WEST;
+        }
```

Comment:
Seems fine (love how the standards change). We did have a formal alias system somewhere in here to handle things like non standard oracle and/or esri WKT.